packages/backend/src/models/GroupsModel.ts

```diff
@@ -221,4 +222,17 @@ export class GroupsModel {
             organizationUuid: updatedGroup.organization_uuid,
         };
     }
+
+    async addUserToGroupIfItExists(userUuid: string, groups: string[]) {
```

Comment:
```suggestion
    async addUserToGroupsIfExist(userUuid: string, groups: string[]) {
```

Reply:
got it!

Reply:
done:  https://github.com/lightdash/lightdash/pull/8508/commits/b5291f0b699f43b57c4aafc273276bae890c44ee